client/src/components/Header.jsx

```diff
@@ -1,23 +1,13 @@
 const Header = () => {
   return (
     <nav className="navbar navbar-inverse">
-      <div className="container-fluid" className="navigation">
-        <div className="navbar-header">
-          <a href="#" className="navbar-brand">UforU</a>
-        </div>
-        <div className="collapse navbar-collapse">
-          <ul className="nav navbar-nav">
-            <li><a href="#">Schools</a></li>
-          </ul>
-          <ul className="nav navbar-nav navbar-right">
-            <li className="dropdown">
-              <a href="#" className="dropdown-toggle" role="button">Settings <span className="caret"></span></a>
-            </li>
-          </ul>
-        </div>
+    <div className="container-fluid">
+      <div className="navbar-header">
+        <a href={global.window.location.origin} className="navbar-brand">UforU</a>
       </div>
+    </div>
     </nav >
```

Comment:
can u fix this extra space in the tag. i know u didnt put it there but you have the power to remove it 💪